test/unit/game_spec.js

```diff
@@ -0,0 +1,16 @@
+var rewire = require("rewire");
+
+describe("TicTacToe", function() {
```

Comment:
Ha. I had refactored that out but clearly reverted, on one of many mistakes, and forgot to check! Will update now.